src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  --box__shadow: 0 1px 8px rgba(61,78,97,0.1);
+  --font__header: avenir, sans-serif;
+  --back__grad: linear-gradient(180deg,#fff,#f6f6f7);
+
+  font-family: var(--font__header);
+}
+
+body {
+  margin-top: 0;
+}
+
+.big-bar {
+  margin: 20px 0;
+  position: relative;
+}
+
+.searchImg {
+  position: absolute;
+  width: 19px;
+  bottom: 26px;
+  left: 26px;
+}
+
+.bigSearchField {
+  box-sizing: border-box;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
+  line-height: 22px;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: var(--box__shadow);
+}
+
+.bigSearchField::placeholder {
+  font-family: var(--font__header);
+}
+
+.bigSearchField :focus {
+  outline: none;
+  background: var(--back__grad);
+  border-radius: 3px;
+}
+
+.little-bar {
+  position: relative;
+}
+
+.searchImgSmall {
+  width: 11px;
+  bottom: 16px;
+  left: 13px;
+}
+
+.smallSearchField {
+  box-sizing: border-box;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  color: #3d4e61;
+  font-weight: 300;
+  line-height: 19px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: var(--box__shadow);
+}
+
+.smallSearchField::placeholder {
+  font-family: var(--font__header);
+}
+
+.smallSearchField :focus {
+  outline: none;
+  background: var(--back__grad);
+  border-radius: 3px;
+}
```

Comment:
Here are still a lot of duplicates. These styles are duplicates of bigSearchInput, so create a class `searchInpunt` and move there all connected styles.

Check examples in [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item